messagix/events.go

```diff
@@ -109,6 +109,12 @@ func (s *Socket) handleReadyEvent(data *Event_Ready) error {
 	data.client = s.client
 	s.client.eventHandler(data.Finish())
 	s.previouslyConnected = true
+
+	s.client.SendMessagesCond.L.Lock()
+	s.client.CanSendMessages = true
+	s.client.SendMessagesCond.Signal()
```

Comment:
Looking at the docs, this seems like it should be Broadcast, not Signal https://pkg.go.dev/sync#Cond.Broadcast